src/components/include/application_manager/application_manager.h

```diff
@@ -136,10 +131,16 @@ class ApplicationManager {
  public:
   virtual ~ApplicationManager() {}
 
+  DEPRECATED
+  virtual bool Init(resumption::LastState&,
+                    media_manager::MediaManager* media_manager) {
+    return false;
```

Comment:
Needs a proper definition

Reply:
@jacobkeeler Done in 4fdae6f97e8445b4aebe5269c4163e028115f508

Reply:
Implementation remains unchanged

Reply:
@jacobkeeler Please take a look https://github.com/smartdevicelink/sdl_core/pull/2626#discussion_r373481596